pkg/core/mempool/hashmap.go

```diff
@@ -141,6 +141,24 @@ func (m *HashMap) Get(txID []byte) transactions.ContractCall {
 	return txd.tx
 }
 
+// Delete a key in the hashmap.
+func (m *HashMap) Delete(txID []byte) {
+	m.lock.Lock()
+	defer m.lock.Unlock()
+
+	var k txHash
+
+	copy(k[:], txID)
+	delete(m.data, k)
```

Comment:
It could be good also to recalculate `m.txsSize`.

```
size := m.data[k]
m.txsSize -= size
```

Reply:
Good catch! Updated it :)